app.py

```diff
@@ -68,35 +72,39 @@ def member_form():
         campaign_id=campaign_id, installment_period=installment_period,
         installments=installments, openended_status=openended_status,
         key=app.config['STRIPE_KEYS']['publishable_key'])
+    """
 
 @app.route('/donate')
 def member2_form():
     form = MemberForm2()
-    if request.args.get('amount'):
-        amount = request.args.get('amount')
-    else:
-        message = "The page you requested can't be found."
-        return render_template('error.html', message=message)
-
-    campaign_id = request.args.get('campaignId', default='')
     installments = 'None'
+    campaign_id = request.args.get('campaignId', default='')
     installment_period = request.args.get('installmentPeriod')
+    amount = request.args.get('amount')
 
     if installment_period is None:
```

Comment:
This could be simplified by removing lines 85 and 86 and changing line 82 to ```installment_period = request.args.get('installmentPeriod', 'monthly')```